db/model/Gdoc/GdocBase.ts

```diff
@@ -115,6 +115,75 @@ export async function getLinkedIndicatorsForCharts(
     })
 }
 
+/**
+ * Helper function to load linked charts for given grapher and explorer slugs.
+ * This is shared between GdocBase.loadLinkedCharts and GdocHomepage.loadAnnouncementLinks
```

Comment:
I'd remove this line. The usage can be inferred from code, and the comment can easily become outdated.